app/Http/Controllers/Auth/SocialiteController.php

```diff
@@ -122,6 +122,8 @@ private function login(string $provider): RedirectResponse
             return redirect()->intended();
         } catch (AuthenticationException $e) {
             toast()->error($e->getMessage());
+
+            return redirect()->route('login')->with('block_auto_redirect', true);
```

Comment:
Yeah, I messed this up a bit..  you need to delete one of these two return statements (either location is the same)